llvm/lib/Target/RISCV/GISel/RISCVCallLowering.cpp

```diff
@@ -89,12 +89,13 @@ struct RISCVOutgoingValueHandler : public CallLowering::OutgoingValueHandler {
                             const MachinePointerInfo &MPO,
                             const CCValAssign &VA) override {
     MachineFunction &MF = MIRBuilder.getMF();
-    uint64_t LocMemOffset = VA.getLocMemOffset();
-
+    uint64_t Offset = 0;
```

Comment:
The spec of this function:

>  The specified value has been assigned to a stack location. Load or store it there, with appropriate extension if necessary.

Can this be anything other than a MemLoc? The spec says it is describing a stack location which I think is always a MemLoc.

Reply:
Yes it can. I have not fully investigated this yet (I did not have the time yet), but if I don't have that if then I will get a std::bad_variant_exception. I will do an investigation and give an update later this week.

Reply:
I have done some debugging and I was able to find out, that it is a reg loc in case of indirect parameter passing.

```
assignValueToAddress info dump BEGIN
ValVT:i32
LocVTi32
regLoc: 1 memLoc: 0 pendingLoc: 0
assignValueToAddress info dump END
```

Reply:
Can this be now removed? It should be.